packages/fast-form-generator-react/src/class-name-contracts/form-item.select.class-name-contract.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Select class name contract
+ */
+export default interface IFormItemSelectClassNameContract {
+    form_item_select_wrapper: string;
```

Comment:
We can just call the wrapper formItemSelect also we can change all of these to formItemSelect_elementName because I believe the intent is that the formItemSelect which is scoped to this file is the block and the HTML element you're referencing should be the element in the BEM syntax